README.md

```diff
@@ -422,3 +483,6 @@ While this approach can provide flexibility and ease of use in some cases, it ca
 * The generated code ensures that only correct types of data are passed between javascript and the API implementation, reducing the overhead of error-prone manual type checking.
 * Enum/dictionary bindings improve language support.
 * Erroneous calls reject early with a `TypeError` and offer developer-friendly error descriptions.
+
+### onprinterstatechange EventHandler for Printer
+Similar to how the `PrintJob` objects provides a subscription for state change events, we could introduce an `onprinterstatechange` to allow tracking printer state changes, effectively utilizing the `Create-Printer-Subscription` IPP functionality. However, since the API is mostly centered around creating & tracking print jobs, we believe it would be sufficient for the developers to query the up-to-date printer state right before submitting a job; another alternative for the developer would be to set up regular calls to `updateAttributes()` via `setInterval()` for selected printers.
```

Comment:
This seems like something to figure out based on developer feedback.

Reply:
I've added a note.